pkg/nsx/services/common/types.go

```diff
@@ -57,8 +57,8 @@ const (
 	TagScopeSubnetCRType            string = "nsx-op/subnet_cr_type"
```

Comment:
Should we rename all CR tags here? eg. `vpc_cr_name`/`ippool_cr_name`/...

Reply:
done